src/Sylius/Behat/Context/Ui/Admin/ManagingTaxonsContext.php

```
@@ -419,4 +420,51 @@ public function iShouldSeeInTheFoundResults(string $taxonName): void
+    {
+        $this->sharedStorage->set('autocomplete_search_results', $this->formElement->searchInParentAutocomplete($taxonName));
+    }
+
+    #[Then('I should see :taxonName in the parent taxon autocomplete results')]
+    public function iShouldSeeInTheParentTaxonAutocompleteResults(string $taxonName): void
+    {
+        /** @var array<string, string> $results */
+        $results = $this->sharedStorage->get('autocomplete_search_results');
+
+        $found = false;
+        foreach ($results as $value => $name) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$value'. (undefined)

(UnusedLocalVariable)